client/src/components/specs/reportPageSpec.js

```diff
@@ -1,12 +1,45 @@
+/* eslint-env jest */
```

Comment:
I would also try to simulate events in each component with `simulate('click')`

Reply:
Added to list to do. Thanks!